plugin/tooling.py

```diff
@@ -161,46 +167,47 @@ def run(self, base_package_name: str) -> None:
         self.view.assign_syntax("Packages/JSON/JSON.sublime-syntax")
         self.writeline("{")
         for k, v in sorted(properties.items()):
-            description = v.get("description")
-            if not isinstance(description, str):
-                description = v.get("markdownDescription")
-            if isinstance(description, str):
-                for line in description.splitlines():
-                    for wrapped_line in textwrap.wrap(line, width=100 - 8 - 3):
-                        self.writeline4('// {}'.format(wrapped_line))
-            else:
-                self.writeline4('// unknown setting')
-            enum = v.get("enum")
-            has_default = "default" in v
-            default = v.get("default")
-            if isinstance(enum, list):
-                self.writeline4('// possible values: {}'.format(", ".join(enum)))
-            if has_default:
-                value = default
-            else:
-                typ = v.get("type")
-                # self.writeline4('// NO DEFAULT VALUE <-- NEEDS ATTENTION')
-                if typ == "string":
-                    value = ""
-                elif typ == "boolean":
-                    value = False
-                elif typ == "array":
-                    value = []
-                elif typ == "object":
-                    value = {}
-                elif typ == "number":
-                    value = 0
+            if isinstance(v, dict):
```

Comment:
Can you instead do?
```suggestion
            if not isinstance(v, dict):
               continue
```
Because the current way increases indentation of the whole block which is not ideal.